uber/templates/registration/duplicate.html

```diff
@@ -0,0 +1,60 @@
+{% extends "base.html" %}{% set admin_area=True %}
+{% block title %}That attendee already has a badge{% endblock %}
+{% block content %}
+<div class="panel panel-default">
+    <div class="panel-body">
+    <h2>This attendee already has a badge!</h2>
+
+      We already have a <a href="form?id={{ attendee.id }}">{{ attendee.full_name }}</a> with email address {{ attendee.email }} in our database with a
+    {% if attendee.paid == c.PAID_BY_GROUP and attendee.group.status == c.WAITLISTED %}
+        waitlisted
+    {% elif attendee.paid == c.HAS_PAID or attendee.paid == c.PAID_BY_GROUP %}
+        paid
+    {% elif attendee.paid == c.NEED_NOT_PAY %}
+        complementary
+    {% elif attendee.paid == c.NOT_PAID %}
+        unpaid
+    {% else %}
+        {{ attendee.paid_label }}
+    {% endif %}
+    {{ attendee.badge_type_label }}
+    {% if attendee.ribbon != c.NO_RIBBON %}
+        ({{ attendee.ribbon_label }})
+    {% endif %}
+    badge{% if attendee.group %}
```

Comment:
Do you need a space after "badge" in case the next line prints? I cannot for certain, but there doesn't seem to be one in front of the paren...